src/wallet.js

```diff
@@ -1985,6 +1995,21 @@ const wallet = {
     }
   },
 
+  /**
+   * Method called when received subscribe_address ws message from full node
+   *
+   * @param {Object} data {success, type, message}
+   *
+   * @memberof Wallet
+   * @inner
+   */
+  onSubscribeAddress(data) {
+    // If an address subscription fails, we stop the service
+    if (data.success === false) {
+      throw new Error(data.message)
```

Comment:
What does it do exactly? Log the error? Stop the wallet?

Reply:
Yes, the wallet is stopped.